includes/price-functions.php

```diff
@@ -56,6 +56,28 @@ function give_get_variable_prices( $form_id = 0 ) {
 
 }
 
+/**
+ * Retrieves the variable price ids for a form
+ *
+ * @since 1.8.8
+ *
+ * @param int $form_id ID of the Give form
+ *
+ * @return array Variable prices
+ */
+function give_get_variable_price_ids( $form_id = 0 ) {
```

Comment:
Looks good, I know we have `give_get_price_id()` but it doesn't retrieve an entire array